target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -95,6 +95,7 @@ tplink,archer-mr200-v6)
 	ucidef_set_led_netdev "lan" "lan" "white:lan" "eth0"
 	ucidef_set_led_netdev "wan" "wan" "white:wan" "wwan0"
 	;;
+mercusys,mb130-4g-v1|\
 tplink,re200-v2|\
 tplink,re200-v3|\
```

Comment:
Move this block upwards to sort alphabetically.

Reply:
Sure, to be updated in the next patch